src/crypto/public_key/prime.js

```diff
@@ -31,8 +31,6 @@ import { getRandomBigInteger } from '../random';
  * @async
  */
 export async function randomProbablePrime(bits, e, k) {
-  const BigInteger = await util.getBigInteger();
```

Comment:
All the functions in this file don't need to be async anymore, AFAICT. Could you drop the `async`s (and `await`s wherever they're called)?